docs/02-app/02-api-reference/01-directives/use-cache.mdx

```diff
@@ -29,11 +29,11 @@ const nextConfig: NextConfig = {
 export default nextConfig
 ```
 
-> The `use cache` directive will be available separately from the`dynamicIO` flag in the future.
+> The `use cache` directive will be available separately from the `dynamicIO` flag in the future.
 
-Caching is a technique to improve the performance of web applications by storing the results of computations or data fetches. In Next.js you can use caching to optimize your applications rendering performance.
+Caching is a technique to improve the performance of web applications by storing the results of computations or data fetches. In Next.js, you can cache results from async data requests, while still enabling dynamic rendering when needed. This allows you optimize your application's rendering performance.
```

Comment:
This section is a bit confusing to me. What's the goal here? Are we trying to explain the `general` cache behaviour in next.js? Would it then not start with `everything is dynamic` by default? This sentence gives me the impression that the default is dynamic rendering is when we want it?

Reply:
You're right Ismael, I was just trying to combine the sentence following Amy's comment. But it's a little more nuance than that, it's dynamic when IO is present. We shouldn't use the word IO, but should explain what triggers dynamic behavior and how `use cache` can be used to make it prerenderable.